rnote-ui/src/main.rs

```diff
@@ -77,3 +82,32 @@ fn setup_windows_env() -> anyhow::Result<()> {
     //std::env::set_var("RUST_LOG", "rnote=debug");
     Ok(())
 }
+
+/// we need to set some env vars for macos app bundles
+#[cfg(target_os = "macos")]
+fn setup_macos_env() -> anyhow::Result<()> {
+    use std::ffi::OsStr;
+    use std::path::{Component, PathBuf};
+
+    let current_dir = std::env::current_dir()?.canonicalize()?;
+    if current_dir
+        .components()
+        .zip(current_dir.components().skip(1))
+        .any(|(a, b)| {
+            if let (Component::Normal(a), Component::Normal(b)) = (a, b) {
+                a == OsStr::new("Contents") && b == OsStr::new("MacOS")
+            } else {
+                false
+            }
+        })
+    {
+        std::env::set_var("XDG_DATA_DIRS", &current_dir.join("/../Resources/share"));
+        std::env::set_var(
+            "GDK_PIXBUF_MODULE_FILE",
+            current_dir.join(PathBuf::from(
+                "/../Resources/lib/gdk-pixbuf-2.0/2.10.0/loaders/loaders.cache",
```

Comment:
This is the wrong path. Lose the `loaders` subdirectory:
```
/../Resources/lib/gdk-pixbuf-2.0/2.10.0/loaders.cache
```

Reply:
ah right, probably copy-pasted the path from the win setup here